src/js/containers/LimitOrder/LimitOrderTable.js

```diff
@@ -827,5 +934,6 @@ export default class LimitOrderTable extends Component {
 LimitOrderTable.propTypes = {
```

Comment:
I dont see this style in our project. If its possible, please update as other files

Reply:
I see. It's from legacy code